src/App/App.js

```diff
@@ -1,28 +1,76 @@
 import React from 'react';
+// import React, { useRef,useState } from 'react';
 import './App.css';
 // import { Utilities } from '../Utilities';
-import PlanetContainer from '../PlanetContainer/PlanetContainer';
+// import PlanetContainer from '../PlanetContainer/PlanetContainer';
+import ReservationPage from '../ReservationPage/ReservationPage';
+import LandingPageOne from '../LandingPageOne/LandingPageOne';
+import LandingPageTwo from '../LandingPageTwo/LandingPageTwo';
+import AboutPage from '../AboutPage/AboutPage';
+
+import {
+  BrowserRouter,
+  Router,
+  Switch, 
+  Nav
+} from "react-router-dom";
+import { Swiper, SwiperSlide } from "swiper/react";
```

Comment:
library dependencies for swiper. 
Swiper - Should live in APP page if navigating between views.